### PR TITLE
Cvx attributes

### PR DESCRIPTION
**Pull request by MariaRohozhkina**

### CODE REVIEW

**Review by TinyRickC137 — APPROVED** [review]

**Review by hardhouse — APPROVED** [review]

**Review by hardhouse — APPROVED** [review]

File: CVX/manual_work/cvx_refresh.sql [hunk 1]
```diff
@@ -0,0 +1,147 @@
+--Backup concept_relationship_manual table and concept_manual table.
```
Comment (Alexdavv) [inline CVX/manual_work/cvx_refresh.sql:1]: @MariaRohozhkina Manual work is not described in the readme file

File: CVX/load_stage.sql [hunk 2]
```diff
@@ -47,10 +47,22 @@ INSERT INTO concept_stage (
 	valid_end_date,
 	invalid_reason
 	)
-SELECT SUBSTR(full_vaccine_name, 1, 255) AS concept_name,
+SELECT CASE 
+		WHEN d.vaccinestatus = 'Non-US'
+			AND d.full_vaccine_name NOT ILIKE '%Non-US%'
```
Comment (Alexdavv) [inline CVX/load_stage.sql:52]: @MariaRohozhkina Why is is NOT like?

Comment (hardhouse) [reply CVX/load_stage.sql:52]: just a refactoring of two lines in https://github.com/OHDSI/Vocabulary-v5.0/pull/776/commits/8af2cfb54dd0ac680dfab3e2a0ac6e49d242f2e1#diff-48bf6ee6d03fe9c21eb548122ff5d43e13ad4a933e9181ae52fd56b9c1f99441R52

File: CVX/manual_work/cvx_refresh.sql (hunk 1 above)
Comment (MariaRohozhkina) [reply CVX/manual_work/cvx_refresh.sql]: Done

File: CVX/manual_work/cvx_refresh.sql [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    target_concept_code varchar(50),
+    target_concept_name varchar(255),
+    target_concept_class_id varchar(50),
+    target_standard_concept varchar(20),
+    target_invalid_reason varchar(20),
+    target_domain_id varchar(50),
+    target_vocabulary_id varchar(50)
+);
+
+--3.3. Review the previous mappings and manually add new to the snomed_mapped table
```
Comment (TinyRickC137) [inline CVX/manual_work/cvx_refresh.sql:60]: Wrong vocab @MariaRohozhkina

**Review by TinyRickC137 — APPROVED** [review]

**Review by hardhouse — APPROVED** [review]

**Pull request merged**